### ISSUE TITLE
Crash on macOS writing minimal odf

### ISSUE THREAD

**Issue by MStraeten** [GrandOrgue/GoOdf#92]
I'm creating a minimum odf via:
1. set organ paramters
2. add manual, rename, set number of keys 32
3. --> Write ODF file is ok:
[Test.organ.txt](https://github.com/GrandOrgue/GoOdf/files/15054802/Test.organ.txt)
4. add a new stop, rename, set number of keys 32
5. --> Write ODF 
--> crash

```
Process 30695 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x6c)
    frame #0: 0x000000010008eba8 GoOdf`Windchestgroup::getIsPercussive(this=0x0000000000000000) at Windchestgroup.cpp:251:9
   248 	}
   249
   250 	bool Windchestgroup::getIsPercussive() {
-> 251 		return m_isPercussive;
   252 	}
   253
   254 	void Windchestgroup::setIsPercussive(bool percussive) {
Target 0: (GoOdf) stopped.
(lldb)
```

Version 0.10.0 was fine

**Comment by larspalo** [comment]
Thanks for reporting. Crash was caused because there is no windchestgroup that can be accessed but the rank/stop is trying anyway. Will be fixed in 0.11.1 soon.

**Comment by MStraeten** [comment]
Fix confirmed, thanks
